server/api/sample_sizes.py

```diff
@@ -99,7 +106,83 @@ def sample_sizes_for_contest(contest: Contest):
 
         else:
             assert election.audit_type == AuditType.HYBRID
-            return {}
+
+            validate_uploaded_manifests(contest)
+            validate_uploaded_cvrs(contest)
+
+            total_ballots = hybrid_contest_total_ballots(contest)
+            vote_counts = hybrid_contest_choice_vote_counts(contest)
+            assert vote_counts
+
+            # Create a stratum for non-CVR ballots
+            non_cvr_contest = sampler_contest.Contest(
+                contest.name,
+                {
+                    "ballots": total_ballots.non_cvr,
+                    "numWinners": typing_cast(int, contest.num_winners),
+                    "votesAllowed": typing_cast(int, contest.votes_allowed),
+                    **{
+                        choice_id: vote_count.non_cvr
+                        for choice_id, vote_count in vote_counts.items()
+                    },
+                },
+            )
+            # In hybrid audits, we only store round contest results for non-CVR
+            # ballots
+            non_cvr_sample_results = (
+                None if round_one else rounds.contest_results_by_round(contest)
+            )
+            non_cvr_stratum = suite.BallotPollingStratum(
+                non_cvr_contest,
+                non_cvr_sample_results,
+                -1,  # TODO what is sample_size supposed to be
+            )
+
+            # Create a stratum for CVR ballots
+            cvr_contest = sampler_contest.Contest(
+                contest.name,
+                {
+                    "ballots": total_ballots.cvr,
+                    "numWinners": typing_cast(int, contest.num_winners),
+                    "votesAllowed": typing_cast(int, contest.votes_allowed),
+                    **{
+                        choice_id: vote_count.cvr
+                        for choice_id, vote_count in vote_counts.items()
+                    },
+                },
+            )
+            cvr_reported_results = rounds.cvrs_for_contest(contest)
+            # The CVR sample results are filtered to only CVR ballots
+            cvr_sample_results = rounds.sampled_ballot_interpretations_to_cvrs(contest)
+            # TODO how to use get_misstatements to create the stratum?
+            cvr_misstatements = suite.get_misstatements(
+                cvr_contest, cvr_reported_results, cvr_sample_results, "???", "???"
```

Comment:
We need to iterate over every winner-loser pair and get the misstatements. So my data type is probably not correct here for elections with more than two candidates. Misstatements should be 

```
{
    (winner1,loser1): {"o1": 0, "o2": 0, "u1": 0, "u2": 0},
    (winner1,loser2): {...},
    ...,
}
```